### PR TITLE
Use new DDR ClassGenerator tool

### PR DESCRIPTION
**Pull request by keithc-ca**
* compile the java source against the generated classes
* merge ddr support makefiles & adjust names of intermediate targets

Requires https://github.com/eclipse/openj9/pull/3099

### CODE REVIEW

File: closed/make/DDR.gmk [hunk 1]
```diff
@@ -0,0 +1,122 @@
...
+		openj9.traceformat \
+	)
+
+DDR_SRC_EXCLUDES := com/ibm/j9ddr/tools/ant
+
+ifeq (,$(filter mz31 mz64,$(OPENJ9_PLATFORM_CODE)))
+DDR_SRC_EXCLUDES += com/ibm/j9ddr/corereaders/tdump
+endif
+
+# Exclude source generated by legacy tool, if present.
```
Comment (pshipton) [inline closed/make/DDR.gmk]: There is no legacy tool for Java 11, is this needed?

Comment (keithc-ca) [reply closed/make/DDR.gmk]: Here, 'legacy' refers to the `PointerGenerator` and `StructureStubGenerator` tools that were in use prior to this change.

Comment (pshipton) [reply closed/make/DDR.gmk]: I'm still not clear on where these legacy classes come from. Are they in git?

Comment (keithc-ca) [reply closed/make/DDR.gmk]: Prior to this change, `DDR-gensrc.gmk` generated Java source into the source tree under `openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/structure` and `openj9/debugtools/DDR_VM/src/com/ibm/j9ddr/vm29/pointer/generated`; if any of those files are around, they need to be ignored.

**Comment by pshipton** [comment]
@keithc-ca please take a look at the build failures

**Comment by keithc-ca** [comment]
Updated for https://github.com/eclipse/openj9/pull/3190.

**Comment by keithc-ca** [comment]
jenkins test all zlinux

File: closed/make/DDR.gmk (hunk 1 above)
Comment (pshipton) [reply closed/make/DDR.gmk]: So basically this handles re-building a pre-existing build with this change added?

Comment (keithc-ca) [reply closed/make/DDR.gmk]: Yes.

**Comment by pshipton** [comment]
This is causing a build failure on AIX https://github.com/eclipse/openj9/issues/3227, reverting until it can be fixed.

**Pull request merged**